src/__tests__/unit/register.controller.unit.ts

```diff
@@ -28,15 +31,30 @@ describe('RegisterController', () => {
     wallet: 'liquality',
     fee: 0.000002,
   });
+  let flyoverPayload = new RegisterPayload({
+    sessionId: '',
+    txHash: '0xc',
+    type: constants.TX_TYPE_PEGOUT,
+    value: 0.005,
```

Comment:
IF we are handling Number type, please ensure that we are using weis (integer) just to avoid precision issues